requirements-dev.txt

```diff
@@ -5,3 +5,4 @@ isort>=5.0
 pycodestyle
 pylint>=2.6.0
 wlc
+pytest~=7.4.3
```

Comment:
Needed??

Reply:
shall I remove this ? I thought of adding `pytest` as any developer would need to run tests.